src/egit.c

```diff
@@ -151,7 +152,7 @@ emacs_value egit_wrap(emacs_env *env, egit_type type, void* data)
 
     // Increase refcounts of owner object(s), if applicable
     switch (type) {
-    case EGIT_COMMIT: case EGIT_TREE: case EGIT_BLOB: case EGIT_TAG: case EGIT_OBJECT:
+    case EGIT_COMMIT: case EGIT_TREE: case EGIT_BLOB: case EGIT_TAG: case EGIT_OBJECT: case EGIT_SIGNATURE:
```

Comment:
Same here, you'll not need to check for `EGIT_SIGNATURE` because it's not necessary to incref anything.